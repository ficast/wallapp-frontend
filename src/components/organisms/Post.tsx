import React, { ReactElement } from "react";
import { ImHome } from "react-icons/im";
import { Link } from "react-router-dom";
import styled from "styled-components";
import theme from "../../theme/nice";

type Post = {
  title: string;
  body: string;
  author: string;
};

export default ({ title, body, author }: Post): ReactElement => {
  return (
    <Container>
      <h1>{title}</h1>
      <p>{body}</p>
      <h3>{author}</h3>
    </Container>
  );
};

const Container = styled.div`
  display: flex;
  justify-content: space-around;
  align-items: center;
  background-color: ${theme.colors.secondary[100]};
`;
