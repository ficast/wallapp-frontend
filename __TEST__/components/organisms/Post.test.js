import React from "react";
import renderer from "react-test-renderer";
import Post from "../../../src/components/organisms/Post";

const title = "";
const body = "";
const author = "";

it("renders correctly", () => {
  const component = renderer
    .create(<Post title={title} body={body} author={author} />)
    .toJSON();
  expect(component).toMatchSnapshot();
});
