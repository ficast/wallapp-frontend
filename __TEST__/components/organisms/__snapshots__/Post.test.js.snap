// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdnylx gUBLOR"
>
  <h3
    className="sc-gtssRu jHTtOQ"
  >
    
  </h3>
  <p
    className="sc-dlnjPT gbPZAM"
  >
    ""
  </p>
  
</div>
`;
